source/catalog.html

```diff
@@ -72,21 +88,34 @@ <h2 class="card__title">неродные просторы</h2>
           </div>
           <div class="card__wrapper">
             <a class="card__like-link" href="#">
-              <img
-                class="card__photo"
-                src="./img/photo-2.png"
-                alt="растительность"
-              />
+              <picture>
+                <source
+                srcset="">
+                <source
+                 srcset="">
+                <img
+                srcset=""
+                  class="card__photo"
+                  src="./img/photo-2.png"
+                  alt="растительность"
+                />
+              </picture>
             </a>
             <h2 class="card__title">Местная растительность</h2>
             <p class="card__descriptions">Автор фото: Сергей</p>
             <div class="card__like-counter">
               <a class="card__like-link" href="#">
-                <img
-                  class="card__like-image"
-                  src="./img/like-icon.svg"
-                  alt="лайк"
-                />
+                <picture>
+                  <source srcset="" />
+                  <source srcset="" />
+
+                  <img
+                    srcset=""
+                    class="card__like-image"
+                    src="./img/like-icon.svg"
+                    alt="лайк"
+                  />
+                </picture>
```

Comment:
Для лайков не нужно.
```suggestion
                  <img
                    class="card__like-image"
                    src="./img/like-icon.svg"
                    alt="лайк"
                  />
```